Samples.xcodeproj/project.pbxproj

```diff
@@ -3441,6 +3470,8 @@
 					AnimateImagesWithImageOverlay,
 					ApplyDictionaryRendererToFeatureLayer,
 					ApplyDictionaryRendererToGraphicsOverlay,
+					ApplyHillshadeRendererToRaster,
+					ApplyHillshadeRendererToRasterView,
```

Comment:
![Screenshot 2025-05-07 at 10 18 38 AM](https://github.com/user-attachments/assets/a13f4da5-a2a7-4a0b-9eb9-192257578f06)

When a resource tag is renamed, it is worth checking if the previous one becomes empty, which can be removed.

```suggestion
```